Services/TodoRepository.cs

```diff
@@ -0,0 +1,62 @@
+using System.Linq.Expressions;
+using Microsoft.EntityFrameworkCore;
+using TodoNET6.db;
+using TodoNET6.Models;
+
+namespace TodoNET6.Services
+{
+    public interface ITodoRepository : IRepository<Todo, Guid> {}
+
+    public class TodoRepository : ITodoRepository
+    {
+        private readonly ApplicationContext _context;
```

Comment:
Can you explain what "ApplicationContext" does/why it is important?

Reply:
ApplicationContext is a part of [EF Core](https://docs.microsoft.com/en-us/ef/core/).

EF Core is one of the ORMs we will use to write queries and interact with the database. It is the C# equivalent of Sequelize.